### DISCUSSION TITLE
Does Renovate support assigning different dependency upgrade PR to different person?

### DISCUSSION THREAD

**Discussion by Devinwong** [renovatebot/renovate#30975] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hi Renovate team,

I only found doc describing adding reviewers/assignees for the whole code repo.
Does Renovate support assigning different dependency upgrade PR to different person?
For example:
If upgrading depA, assign to PersonA.
If upgrading depB, assign to PersonB.

Thanks.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
Yes, you can using `packageRules`. [Example](https://github.com/Rahul-renovate-testing/test-assignees)

**Reply by Devinwong** [reply]
Cool. Thanks!
